components/raftstore-v2/src/operation/command/mod.rs

```diff
@@ -505,10 +507,14 @@ impl<EK: KvEngine, R: ApplyResReporter> Apply<EK, R> {
 
     #[inline]
     pub fn flush(&mut self) {
+        let (index, term) = self.apply_progress();
+        let flush_state = self.flush_state().clone();
```

Comment:
Why clone?

Reply:
Because writebatch is mut borrow next line.